modules/gltf/src/lib/extensions/deprecated/EXT_feature_metadata.ts

```diff
@@ -46,55 +46,49 @@ export function getPropertyTableFromExtFeatureMetadata(
   extension: GLTF_EXT_feature_metadata_GLTF,
   metadataClass?: string
 ): FeatureTableJson | null {
+  /**
+   * Note, 3dTiles is able to have multiple featureId attributes and multiple feature tables.
+   * In I3S we should decide which featureIds attribute will be passed to geometry data.
```

Comment:
Ditto: remove any mention about I3S and 3DTiles. If the function is used only in `tile-converter`, I would move it there

Reply:
Done